EpsilonAuxBMS/Src/syscalls.c

```diff
@@ -0,0 +1,207 @@
+/**

```

Comment:
Why is this file generated for EpsilonAuxBMS? [Dillonsd](https://github.com/Dillonsd) [Trying SPI global IRQ disabled](https://github.com/UCSolarCarTeam/Epsilon-Embedded-Software/commit/4162182b8771c0801f21f009829ae7dde9969208)